### DISCUSSION TITLE
How to show process when git commiting? i.e show husky and linting process

### DISCUSSION THREAD

**Discussion by Dwikavindra** [extrawurst/gitui#1320] [Q&A]
### How to show the process like below when committing from gitui?
I just did a commit with gitui, It kind of bothers me that it doesn't show the process like it's showing below when committing. Any way to enable it so that it shows processes like the image below? I'm kinda clueless when searching through the docs.
![image](https://user-images.githubusercontent.com/71365618/189185138-b8c1f7d6-cf29-4dab-8b28-423a6e98433c.png)

**Comment by extrawurst** [comment]
This is not implemented yet
